### DISCUSSION TITLE
How Can i show a polygon ?

### DISCUSSION THREAD

**Discussion by anis-18** [rnmapbox/maps#3357] [Q&A]
Hello community, please could someone help me how Can i show a polygon in the Map.
I'm receiving same coordinates and i need to show polygon. Please some docs or ressources i'm not able to find how to do it.
I've Seen layers and shapes in docs but i'm not sure if i Can implement polygon with this ? And how Can i provide them with m'y coordinates

**Comment by ronCohen2** [comment]
@anis-18 

`  const ramatGanCoordinates = [
    [34.640041, 31.801056],
    [34.645527, 31.799434],
    [34.645987, 31.795515],
    [34.640632, 31.793837],
    [34.637572, 31.7964],
    [34.640041, 31.801056],
  ];`

>
       <ShapeSource
            id="ramatGanPolygon"
            shape={{
              type: 'Feature',
              geometry: {
                type: 'Polygon',
                coordinates: [ramatGanCoordinates],
              },
            }}>
            <FillLayer
              id="polygonFill"
              style={{
                fillColor: 'rgba(219, 163, 119, 0.1)', // light orange with transparency
                fillOutlineColor: 'rgba(219, 162, 119, 1)', // dark orange for the border
              }}
            />
            <LineLayer
              id="polygonBorder"
              style={{
                lineColor: 'rgba(219, 162, 119, 1)', // light orange with transparency
                lineWidth: 3, // 3px border width
              }}
            />
          </ShapeSource>
